static/app/components/updatedEmptyState.tsx

```diff
@@ -0,0 +1,368 @@
+  'javascript-nextjs': {
+    install: t(
+      'Add Sentry automiatcally to your app with the Sentry Wizard (call this inside your project directory)'
+    ),
+    installCode: 'npx @sentry/wizard@latest -i nextjs',
+  },
+  node: {
+    install: t('Add the Sentry Node SDK as a dependency'),
+    installCode: 'npm install --save @sentry/node',
+    configure: t(
+      "Initialize Sentry as early as possible in your application's lifecycle"
+    ),
+    configureCode:
+      dsn => `// You can also use ESM 'import * as Sentry from "@sentry/node"' instead of 'require'
+const Sentry = require("@sentry/node");
+
+Sentry.init({
+  dsn: "${dsn}",
+  // Performance Monitoring
+  tracesSampleRate: 1.0, //  Capture 100% of the transactions
+});`,
+    sourcemaps: t(
+      'Automatically upload your source maps to enable readable stack traces for Errors.'
+    ),
+    sourcemapsCode: 'npx @sentry/wizard@latest -i sourcemaps',
+    verify: t(
+      'Add this intentional error to your application to test that everything is working right away.'
+    ),
+    verifyCode: `const transaction = Sentry.startTransaction({
+      op: "test",
+      name: "My First Test Transaction",
+});
+
+setTimeout(() => {
+  try {
+    foo();
+  } catch (e) {
+    Sentry.captureException(e);
+  } finally {
+    transaction.finish();
+  }
+}, 99);`,
+  },
+  'python-django': {
+    install: t('Use the following command to install our Python Django SDK'),
+    installCode: "pip install --upgrade 'sentry-sdk[django]'",
+    configure: t(
+      'If you have the Django package in your dependencies, the Django integration will be enabled automatically when you initialize the Sentry SDK. Initialize the Sentry SDK in your Django settings.py file'
+    ),
+    configureCode: dsn => `# settings.py
+import sentry_sdk
+
+sentry_sdk.init(
+  dsn="${dsn}",
+  enable_tracing=True
+)`,
+    verify: t(
+      'Add this intentional error to your application to test that everything is working right away.'
+    ),
+    verifyCode: `# urls.py
+from django.urls import path
+
+def trigger_error(request):
+    division_by_zero = 1 / 0
+
+urlpatterns = [
+    path('sentry-debug/', trigger_error),
+    # ...
+]`,
+  },
+  android: {
+    install: t(
+      'Add Sentry automiatcally to your app with the Sentry Wizard (call this inside your project directory)'
+    ),
+    installCode: 'brew install getsentry/tools/sentry-wizard && sentry-wizard -i android',
+  },
+};
+
+export default function UpdatedEmptyState({project}: {project?: Project}) {
+  const organization = useOrganization();
+  const platformGuidedSteps = project?.platform ? GuidedStepsMap[project.platform] : null;
+
+  const {
+    data: projectKeys,
+    isError: projectKeysIsError,
+    isLoading: projectKeysIsLoading,
+  } = useApiQuery<ProjectKey[]>(
+    [`/projects/${organization.slug}/${project?.slug}/keys/`],
+    {
+      staleTime: Infinity,
+      enabled: !!project,
```

Comment:
very tiny nit: prefer `defined(project)` instead of `!!`, it's just a bit more explicit and readable